special-pages/pages/history/app/components/App.module.css

```diff
@@ -70,6 +80,7 @@ body {
  */
 [data-platform="windows"] {
     .customScroller {
+        overflow-y: auto;
```

Comment:
This prevents the caret buttons from showing when no scrollbar is required. It doesn’t work for Mac.